src/pages-and-resources/PagesAndResources.jsx

```diff
@@ -54,6 +56,12 @@ const PagesAndResources = ({ courseId, intl }) => {
         </div>
 
         <PageGrid pages={pages} />
+
+        <div className="d-flex justify-content-between my-4 my-md-5 align-items-center">
```

Comment:
this is what we'll want to gate behind summary config being available when that API is ready

probably don't need any further gating